tutorials/fiori-ios-scpms-floorplan/fiori-ios-scpms-floorplan.md

```diff
@@ -486,7 +509,7 @@ var loadingIndicator: FUILoadingIndicatorView?
 
 Let's load some data!
 
-We're using the same style we've used in the `SupplierTableViewController`. Implement the following two methods and read the inline comments carefully because you will see that we utilize the `DataQuery` object for making a filter as well as an expand.
+We're using the same style we've used in the `SupplierTableViewController`. Implement the following two methods and read the inline comments carefully because you will see that we utilise the `DataQuery` object for making a filter as well as an expand.
```

Comment:
Please don't use the "royal we" unless you are referring to SAP. When giving instructions to the user, say "you" instead. Please also check the rest ot the tutorial and make necessary changes.